clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -2091,6 +2091,12 @@ bool SemaHLSL::CheckBuiltinFunctionCall(unsigned BuiltinID, CallExpr *TheCall) {
       return true;
     break;
   }
+  case Builtin::BI__builtin_hlsl_wave_active_any_true: {
+    if (SemaRef.checkArgCount(TheCall, 1))
+      return true;
+
+    break;
+  }
```

Comment:
Basic on my understanding, this probably isn't needed (I think because WaveActiveAnyTrue is _always `bool(bool)`?) but I left it in here in case that feeling was incorrect. 

LMK if I should remove it!